### DISCUSSION TITLE
clarificationsFile: matching against any version

### DISCUSSION THREAD

**Discussion by jdddog** [RSeidelsohn/license-checker-rseidelsohn#72] [Q&A]
Hi

When using clarificationsFile, is it possible to match against any version?

Thanks for your help

Jamie

**Comment by RSeidelsohn** [comment]
Hey @jdddog ,

sorry for the late reply - I just saw your question here right now. This feature was not done by me, but I will have a look. Please be aware that I am not often working on this project, but it shoud be at least every 2nd friday from now on, so chances are that I will tackle this issue soon.

Cheers, Roman.
